Yosemite/Yosemite/Tools/POS/PointOfSaleOrderService.swift

```diff
@@ -0,0 +1,219 @@
+import Foundation
+import Networking
+
+public struct PointOfSaleCartProduct {
```

Comment:
❓ We probably want to replace this and its usage with `POSItem` as the latest product representation in the POS.